chi/utils.go

```
@@ -0,0 +1,81 @@
+)
+
+func getRoutes(r chi.Router) []common.PathInfo {
+	var paths []common.PathInfo
+	walkFn := func(method string, route string, _ http.Handler, _ ...func(http.Handler) http.Handler) error {
+		paths = append(paths, common.PathInfo{
+			Method: method,
+			Path:   route,
+		})
+		return nil
+	}
+	chi.Walk(r, walkFn)
```

golangci-lint (1.64.8)

Error return value of `chi.Walk` is not checked

(errcheck)